app/response/utils.py

```diff
@@ -424,6 +456,29 @@ def process_email_template_request(request_id, data):
                                agency_name=agency_name,
                                note_content=note_content,
                                page=page)
+    # process email template for offline instructions
+    if data['type'] == 'instruction_email':
+        # if data['instructions'] exists, use email_content as template with specific instructions template
+        try:
+            instruction = data['instruction']
+            default_content = False
+            content = data['email_content']
+            if instruction['privacy'] != PRIVATE:
+                instruction_content = instruction['content']
+            else:
+                instruction_content = ''
+        # use default_content in response template
+        except KeyError:
```

Comment:
Why are you always expecting a KeyError? It's probably better to use `.get` and check for `None` or just include the default value in the call, depending on your needs. We should talk about this though.